core/src/databases/database.rs

```diff
@@ -91,82 +94,113 @@ pub struct GetTableSchemaResult {
     pub head: Option<Vec<QueryResult>>,
 }
 
-pub async fn get_tables_schema(
+async fn get_local_tables_schema(
```

Comment:
This method is mostly unchanged, just extracted from get_tables_schema and some simplifications